terraform/modules/cluster/eks.tf

```diff
@@ -177,54 +146,5 @@ resource "aws_security_group_rule" "dns_tcp" {
   to_port           = 53
   protocol          = "tcp"
   cidr_blocks       = [module.aws_vpc.vpc_cidr_block]
-  security_group_id = module.eks_blueprints.cluster_primary_security_group_id
-}
-
-# Build kubeconfig for use with null resource to modify aws-node daemonset
-locals {
-  kubeconfig = yamlencode({
-    apiVersion      = "v1"
-    kind            = "Config"
-    current-context = "terraform"
-    clusters = [{
-      name = module.eks_blueprints.eks_cluster_id
-      cluster = {
-        certificate-authority-data = module.eks_blueprints.eks_cluster_certificate_authority_data
-        server                     = module.eks_blueprints.eks_cluster_endpoint
-      }
-    }]
-    contexts = [{
-      name = "terraform"
-      context = {
-        cluster = module.eks_blueprints.eks_cluster_id
-        user    = "terraform"
-      }
-    }]
-    users = [{
-      name = "terraform"
-      user = {
-        token = data.aws_eks_cluster_auth.cluster.token
-      }
-    }]
-  })
-}
-
-resource "null_resource" "kubectl_set_env" {
-  triggers = {
-    cluster_arns = module.eks_blueprints.eks_cluster_arn
-  }
-
-  provisioner "local-exec" {
-    interpreter = ["/bin/bash", "-c"]
-    environment = {
-      KUBECONFIG = base64encode(local.kubeconfig)
-    }
-
-    # Reference docs https://docs.aws.amazon.com/eks/latest/userguide/cni-increase-ip-addresses.html
-    command = <<-EOT
-      sleep 30
-      kubectl set env daemonset aws-node -n kube-system POD_SECURITY_GROUP_ENFORCING_MODE=standard --kubeconfig <(echo $KUBECONFIG | base64 --decode)
```

Comment:
added into the VPC CNI custom configuration so all of this can be removed. Let me know if there is a reason this needs to be done this way instead of the custom config (currently both routes are shown/used)